synthesizer/process/src/cost.rs

```diff
@@ -60,7 +60,12 @@ pub fn deployment_cost<N: Network>(deployment: &Deployment<N>) -> Result<(u64, (
 /// Returns the *minimum* cost in microcredits to publish the given execution (total cost, (storage cost, finalize cost)).
 pub fn execution_cost<N: Network>(process: &Process<N>, execution: &Execution<N>) -> Result<(u64, (u64, u64))> {
     // Compute the storage cost in microcredits.
-    let storage_cost = execution.size_in_bytes()?;
+    let mut storage_cost = execution.size_in_bytes()?;
+
+    // Compute a storage cost penalty if above the size penalty threshold.
+    if storage_cost > EXECUTION_SIZE_PENALTY_THRESHOLD {
+        storage_cost = storage_cost.saturating_mul(100)
```

Comment:
For consistency, I would name the magic 100 number similar to DEPLOYMENT_FEE_MULTIPLIER and put both these new constants in the network trait

Reply:
Yeah the network trait is certainly a better place for this.